content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -292,6 +292,50 @@ When you add a custom resource, you can access it using:
   (generating one is an advanced undertaking, but some projects may provide a client along with
   the CRD or AA).
 
+
+## Custom resource field selectors
+
+[Field Selectors](/docs/concepts/overview/working-with-objects/field-selectors/)
+let clients select custom resources based on the value of one or more resource
+fields.
+
+All custom resources support the `metadata.name` and `metadata.namespace` field
+selectors.
```

Comment:
What about other fields in metadata? Restricted or could be possibly add inside `selectableFields` to make it accessible? Maybe worth to make it more clear?

Reply:
For beta and stable, we should definitely clarify this.

Reply:
Good question. I think I may disallow metadata selectable fields in beta, so I'm going to leave this as-is and circle back in 1.31.